flink-table/flink-table-planner/pom.xml

```diff
@@ -74,24 +59,26 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-shaded-jackson</artifactId>
+			<artifactId>flink-cep_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
 
-		<!-- Used for code generation -->
+		<!-- Table ecosystem -->
```

Comment:
I think we can/should still include `flink-table-common` here, even though it's a transitive dependency.